### ISSUE TITLE
Variables not initialising

### ISSUE THREAD

**Issue by joeyearsley** [hyperdashio/hyperdash-sdk-py#78]
### System:
TF 1.2
Python 3.5
Ubuntu 16.04

### Problem:
We create the preprocessing and model graphs in a script (we'll call it `build_graph.py`) then go to call the actual `tf.run` in another script (we'll call `train func`) at the end of the script.   

The Hyperdash monitor is a decorator for the `train func`, but when we call this script it complains the initial Variables aren't initialised, however, when we run the scripts without hyperdash there are no such complaints.

# TL;DR
Variable initialisation problem

**Comment by richardartoul** [comment]
Hey Joe,

Could you test something for me?

In the decorated train func, could you add the following lines of code:

```python
graph = tf.get_default_graph()
with graph.as_default():
  // Rest of your code
```

**Comment by richardartoul** [comment]
@joeyearsley did that work for you?
